Makefile

```diff
@@ -6,31 +6,36 @@ SHELL := bash
 MAKEFLAGS += --warn-undefined-variables
 MAKEFLAGS += --no-builtin-rules
 MAKEFLAGS += --no-print-directory
-BIN := $(abspath .tmp/bin)
+BIN ?= $(abspath .tmp/bin)
```

Comment:
I can undo this if you want, and probably a lot of other changes in here. I spent a good bit of time trying to get this all happy on Windows (see #166 for the gory details). And many of the changes I made along the way aren't strictly necessary. I left them in since it seemed to provide a little flexibility which could prove useful...

Reply:
Nah just didn't know the context. Thanks.